variables.tf

```diff
@@ -56,12 +56,23 @@ variable "ecr_scan_on_push" {
   description = "Controls if ECR should scan images after pushed."
 }
 
-variable "ecr_number_of_images_to_keep" {
+variable "ecr_number_of_images_to_keep_on_main" {
```

Comment:
Should we change the variable name to `ecr_number_of_images_from_main` to be more concise and a bit shorter?

Reply:
Done